### PR TITLE
change components styling

### LINKED ISSUE #79: Frontend: Login Page
When I want to login or register, I want to be able to use my Slack account so that the app can recognize me as a valid HackYourFuture user.

Create a page below and use the endpoint developed on the issue to show the list #58  and the page component created on this issue #61 -  LoginPage - src/client/containers/LoginPage/LoginPage.component.js

image

IMPORTANT: make sure you use existing components, if needed change the existing components or work together with the developer that created those same components.

### PR DESCRIPTION
**Pull request by nuha90**
# Description

adding flexbox to CSS file

Fixes #79 

# Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Kiran-Serolla — APPROVED** [review]
Looks good

File: src/client/components/SignInButton/Button.js [hunk 1]
```diff
@@ -8,13 +8,15 @@ const Button = (props) => {
...
   };
   return (
-    <button type="submit" className="button" onClick={clicked}>
-      <img
-        alt="Sign in with Slack"
-        src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
-        srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"
-      />
-    </button>
+    <div className="box">
```
Comment (tpetrina) [inline src/client/components/SignInButton/Button.js:11]: `box` is a very generic name and you already are using this class in the footer. Which means that `.box` will affect it.
Also, a `button` shouldn't style its location and/or placement - that is up to the container _around_ it.

**Review by tpetrina — APPROVED** [review]
Looks good!

**Review by 1620prabn — APPROVED** [review]
good job

**Review by Kiran-Serolla — COMMENTED** [review]
good job

**Pull request merged**